doc/DOWNLOADS.md

```diff
@@ -30,9 +30,11 @@ sudo dnf install nicotine+
 
 ### Flatpak
 
-If your distribution supports Flatpak, you can install Nicotine+ from Flathub.
+If your distribution supports [Flatpak](https://www.flatpak.org/setup/), you can install the current stable version of [Nicotine+ from Flathub](https://flathub.org/apps/details/org.nicotine_plus.Nicotine), run the following:
 
-- [Download Nicotine+ on Flathub](https://flathub.org/apps/details/org.nicotine_plus.Nicotine)
+```sh
+flatpak install flathub org.nicotine_plus.Nicotine
```

Comment:
This won't work if the user hasn't added the Flathub repository before.

Reply:
Fixed in https://github.com/nicotine-plus/nicotine-plus/commit/f02090e6edf9eeff0fab2a5f647c759caf37c558